packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -515,7 +515,7 @@ export const HelpCenterContactForm = () => {
 				<section className="contact-form-submit">
 					<Button
 						isBusy={ isFetchingGPTResponse }
-						disabled={ isFetchingGPTResponse }
+						disabled={ isFetchingGPTResponse || isCTADisabled() }
```

Comment:
Can we move `isFetchingGPTResponse` inside `isCTADisabled` to keep all the reasons the button should be disabled in one place?

Reply:
Makes sense, thanks!
While testing again I noticed another issue which was only visible when the debounce timer ended on the form screen - the button would be disabled in the contact form screen, while we were pre-fetching the response (which we don't really need to do, but is out of scope for this fix).